frontend/doc-site/05__Annotate and Analyze Your Data/5_5__Gene Sets.mdx

```diff
@@ -28,7 +28,13 @@ Each row represents a gene in a gene set and must be unique.
 
 Example:
 
-EMBED TABLE
+| gene\_set\_name | gene\_set\_description | gene\_symbol | gene\_description |
+| :--- | :--- | :--- | :--- |
+| club.cell | description | CCKAR | description |
+| club.cell |  | SCGB3A2 | description |
+| club.cell |  | CYP2F2 | description |
+| macrophage | description | CD68 | description |
+| macrophage |  | CD163 | description |
```

Comment:
Viewing these changes on my local frontend, it looks like regular markdown tables aren't compiling properly. Do you think you could swap this over to an HTML table? Here's a generator I found: https://www.tablesgenerator.com/html_tables.  Make sure to select `Do not generate CSS`